packages/hash/frontend/src/pages/_error.page.tsx

```diff
@@ -76,4 +77,6 @@ CustomError.getInitialProps = async ({
   return errorInitialProps;
 };
 
-export default CustomError;
+Page.getLayout = getLayoutWithHeader;
```

Comment:
TODO in a future PR: consider replacing with default layout. Layout header can be throwing too, so we may end up with an infinite cycle of errors.

Reply:
I see. Used LayoutHeader so that it's easy for the user to navigate to the home page by clicking on the Hash logo. Didn't consider LayoutHeader might throw. Will update in a follow up PR